client/src/main/java/org/evosuite/coverage/mutation/MutationObserver.java

```diff
@@ -73,7 +79,14 @@ public static void deactivateMutation() {
 	 * @param mutation a {@link org.evosuite.coverage.mutation.Mutation} object.
 	 */
 	public static void deactivateMutation(Mutation mutation) {
-		activeMutation = -1;
+		activeMutations.clear();
```

Comment:
This is fine, just wondering whether you intended to do `activeMutations.remove(mutation.getId())` or something like that rather than deactivating all mutants? Otherwise we should maybe drop the parameter and rename the method to `deactivateMutations()` ?